src/NzbDrone.Core/MediaFiles/EpisodeImport/ImportDecisionMaker.cs

```diff
@@ -15,7 +17,8 @@ namespace NzbDrone.Core.MediaFiles.EpisodeImport
 {
     public interface IMakeImportDecision
     {
-        List<ImportDecision> GetImportDecisions(List<String> videoFiles, Series series, bool sceneSource, QualityModel quality = null);
+        List<ImportDecision> GetImportDecisions(List<String> videoFiles, Series series);
+        List<ImportDecision> GetImportDecisions(List<String> videoFiles, Series series, bool sceneSource, ParsedEpisodeInfo folderInfo);
```

Comment:
Yeah, I was thinking the name was bad (but didn't want to make 1000 changes and have a nasty PR).